pkg/controller/controller_test.go

```diff
@@ -973,8 +973,8 @@ func TestReconcileInstanceNamespaceError(t *testing.T) {
 	testController.reconcileInstance(instance)
 
 	actions := filterActions(fakeCatalogClient.Actions())
-	if a := len(actions); a != 0 {
-		t.Fatalf("Unexpected number of actions: expected 0, got %v. Actions: %+v", a, actions)
+	if a := len(actions); a != 1 {
```

Comment:
added additional update at L#504 and L#826. Check those. I don't see why they would be a special case to not update.